templates/burger-menu.twig

```diff
@@ -60,23 +60,22 @@
 					{% include "burger-menu-items.twig" with { menu: navbar_menu_items } %}
 				{% endif %}
 
-				{% if donate_menu_items is not empty and donate_menu_items[0].children|length > 1 and dropdown_menu %}
+				{% if donate_menu_items is not empty and donate_menu_items[0].children|length >= 1 and dropdown_menu %}
 					{% include "burger-menu-items.twig" with { menu: donate_menu_items } %}
 				{% endif %}
 			</ul>
 		</nav>
 	</div>
 
-	{% if donate_menu_items|first is not null %}
-		{% set donate_menu = donate_menu_items|first %}
+	{% if donate_menu_default|first is not null %}
```

Comment:
Because of the default values, `donate_menu_default` should never be empty right? Maybe we can remove this check 🤔